### DISCUSSION TITLE
Guide to deploy BunnyCDN?

### DISCUSSION THREAD

**Discussion by shleeable** [mastodon/mastodon#19024] [Q&A]
Hey,

Anybody using BunnyCDN? Could you please write a small guide on using the service? 

I'm using AWS S3 bucket, but I doubt the guide would change much between providers. - I'm going to migrate to BunnyStorage once they release their S3 bucket.

**Comment by Gargron** [comment]
I've not used BunnyCDN for user uploads. Only for static assets. It does help there and pretty straightforward to setup. I think that using a CDN for user uploads would be prohibitely expensive (considering there are videos there as well). I just use nginx as a bandwidth saver as [described in the docs](https://docs.joinmastodon.org/admin/optional/object-storage-proxy/).
